android/app/build.gradle

```diff
@@ -51,6 +51,8 @@ android {
         versionName flutterVersionName
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         multiDexEnabled true
+        testInstrumentationRunner "pl.leancode.patrol.PatrolJUnitRunner"
+        testInstrumentationRunnerArguments clearPackageData: "true"
```

Comment:
what is it?

Reply:
its part of the setup for patrol stated in the [official docs](https://patrol.leancode.co/documentation)